pkg/kubectl/cmd/util/helpers.go

```diff
@@ -394,6 +394,10 @@ func GetPodRunningTimeoutFlag(cmd *cobra.Command) (time.Duration, error) {
 	return timeout, nil
 }
 
+func GetAddressFlag(cmd *cobra.Command) []string {
```

Comment:
these aren't general purpose... remove in favor of direct binding to the opts struct field

Reply:
sure, in fact it originally was direct, but the changes seemed like now this would be the preferred way. Anyway, will revert to previous way.